src/App.tsx

```diff
@@ -8,21 +9,27 @@ import ErrorBoundary from "features/auth/components/ErrorBoundary";
 import { Navigation } from "./Navigation";
 import "./lib/i18n";
 import { WalletProvider } from "features/wallet/WalletProvider";
+import { useServiceWorkerUpdate } from "lib/utils/hooks/useServiceWorkerUpdate";
+
+const CHECK_FOR_UPDATE_INTERVAL = 1000 * 60 * 4;
```

Comment:
I have set the interval to 4 hourly. Happy for feedback on other intervals though.